app/models/project.server.ts

```diff
@@ -651,7 +652,7 @@ export async function searchProjects({
     projectIdsWhere = Prisma.sql`p.id IN (${Prisma.join(
       ids.map((val) => val.id)
     )})`;
-  }
+  }      
```

Comment:
nitpick, but here it shows that there is extra whitespace at the end of this line :-p

Reply:
Fixed